ibm/mas_devops/playbooks/oneclick_add_manage.yml

```diff
@@ -16,7 +16,7 @@
     db2_namespace: db2u
 
     # Application Installation
-    mas_app_id: manage
+    mas_app_id: "{{ lookup('env', 'MAS_APP_ID') }}"
```

Comment:
Can we default this to `manage` when not set please, so it's not a breaking change.

Reply:
sure. done.